evansmusomi/react-native-102
# Configures Input Components and Validates Share Place Form

*What this does*
- Configures keyboard input on auth screen inputs
- Handles soft keyboard on auth screen
- Adds validation to share place form

*How to verify*
- The keyboard displayed when filling in the email address on the auth screen is customised for email
- Pressing within the input container on auth screen (outside input fields) dismisses the keyboard
- The soft keyboard pads the input container to ensure it's always in view on the auth screen
- The share place button is initially disabled, until the place name field is filled in
